packages/website/src/components/switch-link.tsx

```diff
@@ -33,10 +34,12 @@ export type Props = {
 };
 
 const SwitchLink = ({ href, children, router, ...rest }: Props) => {
-  if (!href || href.indexOf('http') === 0 || href.indexOf('#') === 0) {
+  const isExternal = href && href.indexOf('http') === 0;
+  if (!href || isExternal || href.indexOf('#') === 0) {
```

Comment:
This if conditional was intended to be `isExternal` in its entirety - we should probably keep tinkering with this until we're happy that that's true.

Reply:
I've simplified this so we're only checking `isExternal` once and added `target="_blank"`. Let me know if you'd like to discuss further.